src/styles/blocks/card.css

```
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: end;
```

I think "block-stars" and "card__reviews" are center each other
```suggestion
  align-items: center;
```